release_notes/unreleased.md

```diff
@@ -1 +1,2 @@
 **Unreleased**
+* Release on splunk soar [PAPP-21321]
```

Comment:
Update with correct sentence for example "Releasing as open-source Community app"